### DISCUSSION TITLE
[auth] Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by baobach** [community/community#114356] [Copilot Conversations]
### Select Topic Area

Question

### Body

I have opened a ticket and been waiting for 12 hours and still no response from the GitHub support team. Here is my [Ticket](https://support.github.com/ticket/2694178). I have paid for the subscription and still have to experience this slow customer support?

**Comment by ghostinhershell** [comment]
Hi @baobach, 

Thanks so much for reaching out in our community! According to our [GitHub Copilot FAQs + Troubleshooting common issues ⚙️](https://github.com/orgs/community/discussions/47318) post, here's the answer to your question: 

> Due to current trade controls, your GitHub account might be restricted from making use of Copilot. This is a region-based restriction and applies to all users located in some trade-restricted countries, being flagged for the same earlier reasons, and is separate from other GitHub services. Read more about it here: [GitHub and Trade Controls](https://docs.github.com/en/site-policy/other-site-policies/github-and-trade-controls)

> If you feel this should not be the case, feel free to contact our Support team and [open a ticket ](https://support.github.com/request?tags=dotcom-footer)with them.

I know you already mentioned opening a ticket with support. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Reply by hitzhangjie** [reply]
If it's affected by Github and Trace Controls, github could send a mail notification to let users know. I have spent nearly several hours to search similar problems and try to work around this.
